install.html

```diff
@@ -20,30 +19,33 @@
+                <p>Second, install the version of Anaconda which you downloaded, following the instructions on the download page.</p>
 
-                <h3>Installing Jupyter using Anaconda and conda</h3>
-                <p>For new users, we <strong>highly recommend</strong> <a href="https://www.continuum.io/downloads">installing Anaconda</a>. Anaconda conveniently installs Python, the Jupyter Notebook, and other commonly used packages for scientific computing and data science.</p>
-                <p>We recommend using the Anaconda distribution to install Python and Jupyter. We’ll go through its installation in the next section.</p>
-
-                <ul>
-                    <li>Download <a href="https://www.continuum.io/downloads">Anaconda</a>. We recommend downloading Anaconda’s latest Python 3 version (currently Python 3.5).</li>
-                    <li>Install the version of Anaconda which you downloaded, following the instructions on the download page.</li>
-                    <li>Congratulations, you have installed Jupyter Notebook. To run the notebook:
-                    {% highlight bash %}jupyter notebook{% endhighlight %}
-                    </li>
-                </ul>
+                <p>Congratulations, you have installed Jupyter Notebook! To run the notebook, run the following command at 
+                   the Terminal (Mac/Linux) or Command Prompt (Windows): {% highlight bash %}jupyter notebook{% endhighlight %}
+                <p>
                 <p>See <a href="https://jupyter.readthedocs.io/en/latest/running.html#running">Running the Notebook</a> for more details.</p>
 
-                <h3>Alternative for experienced Python users: Installing Jupyter with pip</h3>
-                <p>As an existing Python user, you may wish to install Jupyter using Python’s package manager, <a href="https://jupyter.readthedocs.io/en/latest/glossary.html#term-pip">pip</a>, instead of Anaconda.</p>
+                <h3>Installing Jupyter with pip</h3>
+                <p>As an existing or experienced Python user, you may wish to install Jupyter using Python’s package manager, 
+                    <a href="https://jupyter.readthedocs.io/en/latest/glossary.html#term-pip">pip</a>, instead of Anaconda.
+                </p>
                 <p>First, ensure that you have the latest pip; older versions may have trouble with some dependencies:</p>
                 {% highlight bash %}pip3 install --upgrade pip{% endhighlight %}
                 <p>Then install the Jupyter Notebook using:</p>
                 {% highlight bash %}pip3 install jupyter{% endhighlight %}
-                <p>(Use pip if using legacy Python 2.)</p>
-                <p>Congratulations. You have installed Jupyter Notebook.</p>
+                <p>(If you are running Python 2, change the <code>pip3</code> in these commands to <code>pip</code>.)</p>
```

Comment:
Probably should change the preceding lines.

We recommend using Python 3 for installation. For Python 3:
```
python3 -m pip install --upgrade pip
python3 -m pip install jupyter
```

If you are running Python 2
```
python -m pip install --upgrade pip
python -m pip install jupyter
```